tests/integration/platform/test_routes.py

```diff
@@ -278,3 +278,200 @@ def test_move_section(authenticated_client, factories, db_session):
     section2_from_db = db_session.scalars(select(Section).where(Section.id == section2.id)).one()
     assert section1_from_db.order == 0
     assert section2_from_db.order == 1
+
+
+def test_move_form(authenticated_client, factories, db_session):
+    section = factories.section.create()
+    form1 = factories.form.create(section=section, order=0)
+    form2 = factories.form.create(section=section, order=1)
+    result = authenticated_client.post(
+        url_for(
+            "platform.move_form",
+            grant_id=section.collection_schema.grant.id,
+            collection_id=section.collection_schema.id,
+            section_id=section.id,
+            form_id=form1.id,
+            direction="down",
+        ),
+    )
+    assert result.status_code == 302
```

Comment:
If we're looking for redirects, can we assert on the redirect URL please? Just from reading the test I can't be confident where it's going - I could imagine (from experience sadly) that it might just redirecting us to some login page instead, which would undermine the test.